spec/public-api.yaml

```diff
@@ -995,6 +998,7 @@ components:
       - confidence
       - created_at
       - detector_id
+      - source
```

Comment:
Just noting other conversations we've had, this is unrelated to the goal of this PR, but brings this doc back in sync with the BE